clusterloader2/cmd/clusterloader.go

```diff
@@ -119,6 +119,10 @@ func main() {
 		glog.Fatalf("Config completing error: %v", err)
 	}
 
+	if err = util.LogClusterNodes(f.GetClientSet()); err != nil {
+		glog.Fatalf("Nodes info logging error: %v", err)
```

Comment:
This probably shouldn't fail the test immediately.

Reply:
Changed to `Errorf`. However, I we are not able to list nodes, should it mean that something serious happened?

Reply:
Not necessary - maybe your request just failed because of network issue?